Weather/Weather/Model/ForecastWeatherHourly.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  ForecastWeatherHourtlyModel.swift
+//  Weather
+//
+//  Created by Sergii Lisnyi on 5/18/18.
+//  Copyright © 2018 Sergii Lisnyi. All rights reserved.
+//
+
+import Foundation
+
+struct ForecastWeatherHourly {
+    var city: String?
+    var tempCurrent: String?
+    var arrayWeatherHourly = [(time: String, temp: String)]()
```

Comment:
Try not to use name of the collection type (array, dictionary, set etc) until you really need this.
Also, I would suggest to use struct for hourly entries